www/src/pages/components/cards.js

```diff
@@ -102,6 +103,15 @@ export default function CardSection({ data }) {
       <p />
       <ReactPlayground codeText={CardImgOverlay} />
 
+      <h3>
+        <Anchor id="card-example-layout">Card layout</Anchor>
+      </h3>
```

Comment:
As you're going here, might be worth rebaseing with v4-support and using the new `Heading` and `ComponentApi` components in the docs instead of Anchor and PropTable

Reply:
It will still be open. you can do whatever you want to the branch and the pr will update, and force pushing to your own branch is fine